server/store/middleware/infohash/whitelist_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2016 The Chihaya Authors. All rights reserved.
+// Use of this source code is governed by the BSD 2-Clause license,
+// which can be found in the LICENSE file.
+
+package infohash
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/chihaya/chihaya"
+	"github.com/chihaya/chihaya/tracker"
+)
+
+func TestWhitelistAnnounceMiddleware(t *testing.T) {
+	var (
+		achain tracker.AnnounceChain
+		req    chihaya.AnnounceRequest
+		resp   chihaya.AnnounceResponse
+	)
+	achain.Append(whitelistAnnounceInfohash)
+	handler := achain.Handler()
+
+	err := handler(nil, &req, &resp)
+	assert.Equal(t, ErrBlockedInfohash,err)
+
+	req.InfoHash = chihaya.InfoHash("def")
+	err = handler(nil, &req, &resp)
+	assert.Equal(t, ErrBlockedInfohash, err)
+
+	req.InfoHash = chihaya.InfoHash("abc")
+	err = handler(nil, &req, &resp)
+	assert.Nil(t, err)
+}
+
+func TestWhitelistScrapeMiddleware(t *testing.T) {
+	var (
+		schain   tracker.ScrapeChain
+		mwconfig = chihaya.MiddlewareConfig{
+			Name: "whitelist_infohash",
+			Config: Config{
+				Mode: ModeBlock,
+			},
+		}
+		req     chihaya.ScrapeRequest
+		resp    chihaya.ScrapeResponse
+		handler tracker.ScrapeHandler
+		mw      tracker.ScrapeMiddleware
+		err     error
+	)
+
+	// block
+	mw, err = whitelistScrapeInfohash(mwconfig)
```

Comment:
by making this `:=` you can get rid of the last two variables in the `var` block